tests/Drupal/InfoFiles/drupal8/test.info.yml

```diff
@@ -5,3 +5,5 @@ name: Test
 project: test
 datestamp: 1234567
 version: 1.2
+dependency:
+  drupal.taxonomy
```

Comment:
I think we should start a new info.yml test file in the drupal8 folder, then we don't mix test cases as much and they can be isolated better from each other. Please create a dependencies_array.info.yml test file.

Reply:
I will create a new file for this.